ggmap/analyses.py

```diff
@@ -4082,6 +4082,144 @@ def post_cache(cache_results):
                      array=1,
                      **executor_args)
 
+def ancom(counts: pd.DataFrame, rank, taxonomy: pd.Series, grouping: pd.Series, min_mean_abundance_per_feature: float=0.005,
+          ppn=1, pmem='4GB', **executor_args):
+    """Execute ANCOM analysis through Qiime2.
+
+    Paramaters
```

Comment:
Parameter description for grouping is missing.

Reply:
added with latest commit